juniper_official/Kernel/README.md

```diff
@@ -383,3 +383,66 @@ jlock hogs or TCP connection drop messages get reported in these log files
 around the same time. Report to Juniper if this is happening consistently
 or if there is a noticeable side effect seen (such as protocol flaps / traffic
 loss) around the same time.
+
```

Comment:
This file is conflicting, please resolve issue

Reply:
This README.md was added from my commit. But in the latest my README is not present. You can treat this README as a new file. Talked to Anchal, dance resolving the comment.